pkg/platform/kube/platform.go

```diff
@@ -1118,32 +1099,6 @@ func (p *Platform) GetExternalIPAddresses() ([]string, error) {
 		return externalIPAddress, nil
 	}
 
-	nodes, err := p.GetNodes()
-	if err != nil {
-		return nil, errors.Wrap(err, "Failed to get nodes")
-	}
-
```

Comment:
"via" support is removed in this PR, remove it from from function docstring too (Line 1087)